metals/src/main/scala/scala/meta/internal/builds/MillBuildTool.scala

```diff
@@ -42,7 +43,8 @@ case class MillBuildTool(userConfig: () => UserConfiguration)
     val millWrapper =
       if (Properties.isWin) "millw.bat"
       else "millw"
-    val out = BuildTool.copyFromResource(tempDir, millWrapper)
+    val out =
+      BuildTool.copyFromResource(RelativePath(".metals").toNIO, millWrapper)
```

Comment:
Shouldn't we use:
```suggestion
      BuildTool.copyFromResource(workspace.resolve(".metals").toNIO, millWrapper)
```

Relative path should probably start from `.`, but that might not always be the case.

Reply:
Ah yea, good call. Switched it.